app/Notifications/Training/WaitingListRemovalAdded.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace App\Notifications\Training;
+
+use App\Notifications\Notification;
+use Carbon\Carbon;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Notifications\Messages\MailMessage;
+
+class WaitingListRemovalAdded extends Notification implements ShouldQueue
+{
+    use Queueable;
+
+    private $list_name;
```

Comment:
Since we are using PHP 8, I'd suggest type-hinting these class variables now

REF: https://www.phpclasses.org/blog/post/1047-php-8-type-hinting.html#th_types_prop